pkg/math/oracle/aggregator.go

```diff
@@ -1,79 +1,209 @@
 package oracle
 
 import (
-	"maps"
+	"fmt"
```

Comment:
No new code, just renaming and moving to a different file name since it was previously in median.go